catalog/imaging/imaging-app/src/main/resources/features.xml

```diff
@@ -28,7 +28,7 @@
         <bundle dependency="true">mvn:org.codice.alliance.catalog.core/catalog-core-api/${project.version}</bundle>
         <bundle>mvn:org.codice.alliance.imaging/imaging-nitf-api/${project.version}</bundle>
         <bundle>mvn:org.codice.alliance.imaging/imaging-nitf-impl/${project.version}</bundle>
-        <bundle>mvn:org.codice.alliance.imaging/imaging-service-api/${project.version}</bundle>
+        <bundle dependency="true">mvn:org.codice.alliance.imaging/imaging-service-api/${project.version}</bundle>
```

Comment:
You intend for your feature to only be deployable to an environment already includes this api, correct? You do not intend this KAR to be hot-deployable to DDF (or any other Karaf server without Alliance features installed)?